node/network/protocol/Cargo.toml

```diff
@@ -13,7 +13,7 @@ polkadot-node-jaeger = { path = "../../jaeger" }
 parity-scale-codec = { version = "3.1.5", default-features = false, features = ["derive"] }
 sc-network = { git = "https://github.com/paritytech/substrate", branch = "master" }
 sc-authority-discovery = { git = "https://github.com/paritytech/substrate", branch = "master" }
-strum = { version = "0.24", features = ["derive"] }
+strum = { version = "0.24.1", features = ["derive"] }
```

Comment:
?

Reply:
got error when running ```cargo update -p sp-io```, that was indicating that the version needs to be ```^0.24.1```

Reply:
this solved the error.

Reply:
just realized there is no need for this.